includes/Modules/Sign_In_With_Google.php

```diff
@@ -321,7 +321,7 @@ private function render_signin_button() {
 			// Purposely not translated as this is a technical comment.
 			//
 			// See: https://github.com/google/site-kit-wp/pull/9826#discussion_r1876026945.
-			"\n<!-- Sign in with Google button added by Site Kit -->\n"
+			'\n<!-- Sign in with Google button added by Site Kit -->\n'
```

Comment:
The double-quotes here are required or the newline won't be output unescaped:

![Screenshot 2024-12-23 at 16 43 52](https://github.com/user-attachments/assets/82b40724-588a-4ceb-b011-7a05187e078e)
![Screenshot 2024-12-23 at 16 44 05](https://github.com/user-attachments/assets/64e127f9-cb9b-47fd-95dd-05c1707f129f)
```suggestion
			"\n<!-- Sign in with Google button added by Site Kit -->\n"
```